pkg/config/sync.go

```diff
@@ -317,6 +317,9 @@ func mergeConfig(from, to *controller.OperatorConfiguration) {
 			templateSpecContentCopy := from.Workspace.DefaultTemplate.DeepCopy()
 			to.Workspace.DefaultTemplate = templateSpecContentCopy
 		}
+		if from.Workspace.DefaultSchedulerName != "" {
+			to.Workspace.DefaultSchedulerName = from.Workspace.DefaultSchedulerName
+		}
```

Comment:
Please also add a section to log if default scheduler name is set in `GetCurrentConfigString`